controllers/util/k8sutil.go

```diff
@@ -92,6 +94,16 @@ func GetPodNamespace() string {
 	return podNamespace
 }
 
+var csiCephUserCurrGen = os.Getenv(CsiCephUserCurrGenEnvVar)
```

Comment:
Why do we need this a global value? There should not be any case where we should relay on a global generation
Also use, Generation not Gen